stdlib/math.pkl

```diff
@@ -145,6 +145,11 @@ external function acos(x: Number): Float
 /// [1]: https://en.wikipedia.org/wiki/Inverse_trigonometric_functions
 external function atan(x: Number): Float
 
+/// Returns the [4-quadrant inverse tangent][1] of [x] and [y].
+///
+/// [1]: https://en.wikipedia.org/wiki/Atan2
```

Comment:
Since we're linking to Wikipedia, let's use verbiage that aligns with it.

Also, need a `@Since` annotation.

```suggestion
/// Returns the [2-argument arctangent][1] of [x] and [y].
///
/// [1]: https://en.wikipedia.org/wiki/Atan2
@Since { version = "0.28.0" }
```

It might also be helpful to have one more paragraph to describe this function's purpose. Maybe something like: "The result is `θ` (theta) when converting cartesian coordinates [x] and [y] to polar coordinates `r` and `θ`."